### PR TITLE
Improve profile page

### PR DESCRIPTION
**Pull request by nicolaskempf57**
Closes https://github.com/datagouv/front-end/issues/177
Parts of https://github.com/datagouv/front-end/issues/38

### CODE REVIEW

File: components/User/AdminUserProfilePage.vue [hunk 1]
```diff
@@ -80,6 +80,25 @@
         :label="$t('Website')"
         type="url"
       />
+      <UploadGroup
+        :label="$t('Profile picture')"
+        type="drop"
+        accept=".jpeg, .jpg, .png"
+        :hint-text="$t('Max size: 4Mo. Accepted formats: JPG, JPEG, PNG')"
```
Comment (maudetes) [inline components/User/AdminUserProfilePage.vue:87]: Is the max size actually applied?

File: components/User/AdminUserProfilePage.vue [hunk 2]
```diff
@@ -278,8 +298,34 @@ const passwordId = useId()
...
+  if (!profilePicture.value) return null
+  if (typeof profilePicture.value === 'string') return profilePicture.value
+  return URL.createObjectURL(profilePicture.value)
+})
+
 async function updateMe() {
   loading.value = true
+  if (profilePicture.value) {
+    try {
+      await uploadProfilePicture(profilePicture.value)
```
Comment (maudetes) [inline components/User/AdminUserProfilePage.vue:323]: Do we want to update the avatar at the top of the page upon upload?

**Review by maudetes — APPROVED** [review]
:rocket:

File: components/User/AdminUserProfilePage.vue (hunk 2 above)
Comment (nicolaskempf57) [reply components/User/AdminUserProfilePage.vue:323]: It should update both the avatar on the admin page and small one inside the header

File: components/User/AdminUserProfilePage.vue (hunk 1 above)
Comment (nicolaskempf57) [reply components/User/AdminUserProfilePage.vue:87]: This is not a native thing we can do like the formats, so it's never applied on the front-end.

But we could / should work on a better image upload experience with cropping like https://pqina.nl/filepond/

File: components/User/AdminUserProfilePage.vue (hunk 2 above)
Comment (maudetes) [reply components/User/AdminUserProfilePage.vue:323]: Indeed, it is a case after a reinstall!

**Pull request merged**